hwcomposer/qeglfsscreen.cpp

```diff
@@ -44,11 +44,27 @@
 
 #include <private/qmath_p.h>
 
+#ifdef WITH_SENSORS
+#include <QtSensors/QSensorManager>
+#include <QtSensors/QOrientationSensor>
+#include <QtSensors/QOrientationFilter>
+#include <QtSensors/QOrientationReading>
+#include <qpa/qwindowsysteminterface.h>
+
+#include <QTimer>
+
+#include "moc_qeglfsscreen.cpp"
```

Comment:
Are moc files added usually like this? I thought those are automatically handled by qmake?